src/backend/CMakeLists.txt

```diff
@@ -83,6 +85,8 @@ list(APPEND BACKEND_SOURCES
     ${OPENGL_DIR}/CommandBufferGL.h
```

Comment:
nit: move nxt::opengl::DepthStencilState in its own file, DepthStencilStateGL.h/cpp